Dockerfile

```diff
@@ -4,7 +4,10 @@ ADD etc/nodesource.gpg.key /etc
 
 WORKDIR /tmp
 
-RUN yum -y install gcc-c++ && \
+RUN yum -y install gcc-c++ \
+        findutils \
+		zip \
+		aws-cli && \
```

Comment:
This block has tabs and spaces mixed up. Best practice is to start on a new line for multi-argument invocations of package installers like yum and sort the packages alphabetically.